quick_tour/the_view.rst

```diff
@@ -160,7 +160,7 @@ Don't forget to check out the official `Twig documentation`_ to learn everything
 about filters, functions and tags.
 
 Including other Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------
```

Comment:
The original `~~~` is correct in this case because this is a third-level heading (`---` is second level).